web/src/app/profile/BountyDetails/BountyDetails.js

```diff
@@ -119,15 +121,50 @@ class BountyDetails extends Component {
       return this.getAcceptButtons();
     } else if (curBounty.state === OPEN) {
       if (this.props.isStreamer) {
-        return this.getOpenButtons();
+        return curBounty.userVote === COMPLETED ? this.getMarkedCompleted() : this.getOpenButtons();
       } else if (this.props.isDonor) {
-        return this.getAcceptButtons();
+        return curBounty.userVote === FAILED ? this.getMarkedFailed() : this.getAcceptButtons();
       }
     }
 
     return '';
   }
 
+  getMarkedCompleted() {
+    const tooltip = (
+      <Tooltip id="tooltip">
+        You have marked this bounty complete. This bounty will be completed when voting has finished
+      </Tooltip>
+    );
+
+    return (
+      <OverlayTrigger placement="bottom" overlay={tooltip}>
+        <div className={cx(styles.rightButtons, styles.voted)}>
+
+            <div className={styles.checkmark}><Glyphicon glyph="ok" /></div>
+            <div className={styles.text}>Marked Completed</div>
+        </div>
+      </OverlayTrigger>
+    );
+  }
+
+  getMarkedFailed() {
+    const tooltip = (
+      <Tooltip id="tooltip">
+        You have marked this bounty failed. This bounty will be completed when voting has finished
```

Comment:
Do we know who is voting at this point? Can we have custom messaging based on if its the streamer/donator doing the vote?

Like if the Donator votes its completed, we can have messaging saying that "this will complete the contract and release funds to streamer."

If Streamer votes completed we would say we can have messaging saying what you have here "This bounty will be completed when voting has finished"